app/models/manageiq/providers/ibm_cloud/inventory/collector/power_virtual_servers.rb

```diff
@@ -38,4 +38,23 @@ def ports(network_id)
   def sshkeys
     connection.get_ssh_keys
   end
+
+  def system_pool
+    connection.get_system_pool.values
+  end
+
+  def storage_types
+    connection.get_storage_types[region]
+  end
+
+  def region
+    manager.provider_region || "us-south"
+  end
+
+  def guid
+    manager.uid_ems
+  end
+
+  def refresh
+  end
```

Comment:
This doesn't look like it is used either

Reply:
was existing already. Don't know why it was put. Going to remove it also.

Reply:
Maybe a merge conflict thing?  It looks like it is being added by your commit

Reply:
yes, maybe